CMP1903_A1_2324/Die.cs

```diff
@@ -8,16 +8,12 @@ namespace CMP1903_A1_2324
 {
     internal class Die
     {
-        /*
-         * The Die class should contain one property to hold the current die value,
-         * and one method that rolls the die, returns and integer and takes no parameters.
-         */
-
-        //Property
-
-
-        //Method
-
+        static Random random = new Random(); //importing use of random into project
```

Comment:
```suggestion
        static Random random = new Random(); // Instantiating a new random object

Reply:
This line is creating a new random object rather than what python does and import the random module